test/core.jl

```diff
@@ -0,0 +1,38 @@
+@testset "Core" begin
+
+    points = [Point(1,1), Point(2,2), Point(3,3)]
+    img = zeros(Gray{N0f8}, 3, 3)
+    draw!(img, points)
```

Comment:
we may want to change these tests to 

```julia
@inferred draw!(img, points)
```

in the future to spot obvious inference issues. I don't think this needs to be done right now though